scout/server/blueprints/cases/controllers.py

```diff
@@ -526,40 +527,46 @@ def update_synopsis(store, institute_obj, case_obj, user_obj, new_synopsis):
         store.update_synopsis(institute_obj, case_obj, user_obj, link, content=new_synopsis)
 
 
-def update_individuals(store, institute_obj, case_obj, user_obj, ind, age, tissue):
-    """Handle update of individual data (age and/or Tissue type) for a case"""
-
-    case_individuals = case_obj.get("individuals")
-    for subject in case_individuals:
-        if subject["individual_id"] == ind:
-            if age:
-                subject["age"] = round(float(age), 1)
-            else:
-                subject["age"] = None
-            if tissue:
-                subject["tissue_type"] = tissue
-
-    case_obj["individuals"] = case_individuals
-    # update case with new individual data
+def _update_case(store, case_obj, user_obj, institute_obj, verb):
+    """ Update case with new sample data, and create an associated event """
     store.update_case(case_obj, keep_date=True)
 
-    # create an associated event
     link = url_for(
         "cases.case",
         institute_id=institute_obj["_id"],
         case_name=case_obj["display_name"],
     )
+
     store.create_event(
         institute=institute_obj,
         case=case_obj,
         user=user_obj,
         link=link,
         category="case",
-        verb="update_individual",
```

Comment:
This must have been a bug!

Reply:
Well, lets perhaps say it would have been neater if there was only one update-individuals/samples function called by both views, but as it is both exist. I guess it was the distinction of why one typically has multiple samples between cancer and rd that drove the duplication. 🤷